onnxruntime/python/tools/quantization/onnx_quantizer.py

```diff
@@ -446,6 +447,17 @@ def _get_dynamic_input_quantization_params_uint8(self, input_name, nodes_list):
         input_scale_name = input_name + "_scale"
         input_zp_name = input_name + "_zero_point"
 
+        # Add tensors for quantize range and zero value.
+        initializer_qrange = onnx.helper.make_tensor(
+            self.fixed_qrange_uint8_name,
+            onnx_proto.TensorProto.FLOAT,
+            [],
+            [get_qrange_for_qType(qType)],
+        )
+        self.model.add_initializer(initializer_qrange)
+        initializer_qvalue = onnx.helper.make_tensor(self.fixed_zero_name, onnx_proto.TensorProto.FLOAT, [], [0.0])
+        self.model.add_initializer(initializer_qvalue)
+
```

Comment:
newbie question, would this function be called multiple times? If so would this creates multiple copies of the same initializer?